.github/workflows/windows-msys2-build.yml

```diff
@@ -0,0 +1,98 @@
+name: Windows/MSYS2 Build
+on: [workflow_dispatch]
+
+jobs:
+  msys2-mingw64:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+      - name: Checkout Geany-Plugins
+        uses: actions/checkout@v4
+      - name: Checkout Geany
+        uses: actions/checkout@v4
+        with:
+          repository: giuspen/geany
+          path: .geany_source
+      - name: Setup MSYS2
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install: |
+            mingw-w64-x86_64-gcc
+            mingw-w64-x86_64-autotools
+            mingw-w64-x86_64-gtk3
+            mingw-w64-x86_64-python-docutils
+            mingw-w64-x86_64-check
+            mingw-w64-x86_64-cppcheck
+            mingw-w64-x86_64-ctpl-git
+            mingw-w64-x86_64-enchant
+            mingw-w64-x86_64-gpgme
+            mingw-w64-x86_64-gtkspell3
+            mingw-w64-x86_64-libgit2
+            mingw-w64-x86_64-libsoup3
+            mingw-w64-x86_64-lua51
+            patch
+            rsync
+            unzip
+            dos2unix
+            mingw-w64-x86_64-nsis
+      - name: GTK-Bundle-Geany
+        run: |
+          mkdir -p geany_build/bundle/geany-gtk
+          cd geany_build/bundle/geany-gtk
+          bash $(cygpath -u "${GITHUB_WORKSPACE}")/.geany_source/scripts/gtk-bundle-from-msys2.sh -3
+      - name: GTK-Bundle-Geany-Plugins
+        run: |
+          mkdir -p geany_build/bundle/geany-plugins-dependencies
+          cd geany_build/bundle/geany-plugins-dependencies
+          bash $(cygpath -u "${GITHUB_WORKSPACE}")/build/gtk-bundle-from-msys2.sh -3
+      - name: CI-Build-Geany
+        run: |
+          DESTINATON=$(cygpath -u "${GITHUB_WORKSPACE}")/geany_build
+          cd .geany_source
+          VERSION=$(autom4te --no-cache --language=Autoconf-without-aclocal-m4 --trace AC_INIT:\$2 configure.ac)
+          NOCONFIGURE=1 ./autogen.sh
+          export lt_cv_deplibs_check_method=${lt_cv_deplibs_check_method='pass_all'}
```

Comment:
what's this about?

Reply:
> what's this about?

I remember I was finding an error, googled for it and found this as the solution.
I later noticed that Enrico did the same, see https://github.com/geany/geany-plugins/blob/b95b09c124c24c12046dd2660b53821affd6b73d/build/ci_mingw64_geany_plugins.sh#L79